src/System.Runtime.WindowsRuntime/src/System/Threading/WindowsRuntimeSynchronizationContext.cs

```diff
@@ -198,8 +191,8 @@ public void Invoke()
                 // If there was an ETW event that fired at the top of the winrt event handling loop, ETW listeners could
                 // use it as a marker of completion of the previous request. Since such an event does not exist we need to
                 // fire the "done handling off-thread request" event in order to enable correct work item assignment.
-                if (FrameworkEventSource.Log.IsEnabled(EventLevel.Informational, FrameworkEventSource.Keywords.ThreadTransfer))
-                    EtwFireThreadTransferReceiveHandledObj(this);
+                if (FrameworkEventSourceSupport.IsEnabled(EventLevel.Informational, FrameworkEventSource.Keywords.ThreadTransfer))
+                    FrameworkEventSourceSupport.ThreadTransferReceiveHandledObj(this, 3, string.Empty);
```

Comment:
What tool is paying attention to these ETW events?  I'm wondering if either a) we could just delete this code, or b) if we could have an EventSource in this library rather than calling into one exposed from corelib?

Reply:
@vancem, perfview use these ETW events. see https://github.com/dotnet/coreclr/pull/18423